pkg/storage/tableland/impl/sqlite/store.go

```diff
@@ -54,8 +54,8 @@ func (s *Store) InsertParts(ctx context.Context, parts []local.Part) error {
 }
 
 // InsertLayers implements InsertLayers.
-func (s *Store) InsertLayers(ctx context.Context, cid string, layers []local.Layer) error {
-	sql, err := s.factory.SQLForInsertingLayers(tableland.LayersDefinition.Prefix, cid, layers)
+func (s *Store) InsertLayers(ctx context.Context, layers []local.Layer) error {
+	sql, err := s.factory.SQLForInsertingLayers(tableland.LayersDefinition.Prefix, layers)
```

Comment:
Same thing, imagery cid now stored in the lookups table, so not needed in this file anymore.

Reply:
Plus some other cleanup.